marlowe-playground-client/src/Home.purs

```diff
@@ -3,42 +3,87 @@ module Home where
-        [ div [ classes [ ClassName "text-block" ] ]
-            [ h2_ [ text "What is Marlowe?" ]
-            , p_ [ text "Marlowe is special-purpose language for financial contracts on Cardano, allowing contracts to be written in the language of finance, rather than using a general-purpose language on the blockchain. Because it is special-purpose, it is easier to read, write and understand Marlowe contracts. It is also safer: some sorts of errors are impossible to write, and we can completely analyse contract behaviour without having to run a contract." ]
+  div [ classNames [ "flex", "flex-col", "items-center" ] ]
+    [ h1 [ classNames [ "font-semibold", "text-4xl", "my-16" ] ] [ text "Get started" ]
+    , div [ classNames [ "mb-6" ] ]
+        [ button
+            [ classNames (secondaryButton <> [ "mr-small", "w-56", "text-base" ])
+            , onClick ((const <<< Just <<< OpenModal) NewProject)
             ]
-        , div [ classes [ ClassName "text-block" ] ]
-            [ h2_ [ text "What is the Marlowe Playground?" ]
-            , p_ [ text "In the browser-based Marlowe Playground you can write Marlowe contracts, in a variety of different ways. Once a contract is written, you can analyse its behaviour, e.g. checking whether any payments made by the contract could conceivably fail. You can also step through how a contract will behave, simulating the actions of the participants, and read a comprehensive tutorial on Marlowe and the Playground." ]
-            ]
-        , div [ classes [ ClassName "text-block" ] ]
-            [ h2_ [ text "How does the playground work?" ]
-            , p_ [ text "Marlowe contracts can be built in different ways. You can write them as Marlowe text, but also use the Blockly visual programming tool to create contracts by fitting together blocks that represent the different components. Marlowe is embedded in JavaScript and Haskell, and so you can use features from them to help you to build Marlowe contracts more readably and succinctly." ]
+            [ text "Open existing project" ]
+        , button
+            [ classNames (primaryButton <> [ "ml-small", "w-56", "text-base" ])
+            , onClick ((const <<< Just <<< OpenModal) OpenDemo)
             ]
+            [ text "Open an example" ]
+        ]
+    , span [ classNames [ "text-base", "mb-4" ] ] [ text "Or" ]
+    , div [ classNames [ "mb-16", "text-base" ] ]
+        [ span [ classNames [ "font-bold" ] ] [ text "Choose" ]
+        , span_ [ text " a starting environment that's right for you." ]
         ]
-    , hr_
-    , div [ classes [ flex, ClassName "start-with-container" ] ]
-        [ div [ classes [ ClassName "rectangle" ] ] [ img [ src option1 ] ]
-        , div [ classes [ ClassName "rectangle" ] ] [ img [ src option2 ] ]
-        , div [ classes [ ClassName "rectangle" ] ] [ img [ src option3 ] ]
+    , div
+        [ classNames [ "flex", "mb-8" ] ]
+        [ div
```

Comment:
tiny thing, but I feel this and the other other elements like it should be `<a>`s

Reply:
I think an `a` tag can't contain an `img` tag.